test/alternative_ad_frontend.jl

```diff
@@ -34,8 +50,15 @@ dup = Zygote.gradient(senseloss(InterpolatingAdjoint()), u0p)[1]
 
 @test ForwardDiff.gradient(senseloss(InterpolatingAdjoint()), u0p) ≈ dup
 
-struct senseloss2
-    sense::Any
+@test Enzyme.gradient(Reverse, senseloss(InterpolatingAdjoint()), u0p) ≈ dup
+@test_broken Enzyme.gradient(Reverse, senseloss(ReverseDiffAdjoint()), u0p) ≈ dup
+@test Enzyme.gradient(Reverse, senseloss(TrackerAdjoint()), u0p) ≈ dup
+@test Enzyme.gradient(Reverse, senseloss(ForwardDiffSensitivity()), u0p) ≈ dup
+@test_throws SciMLSensitivity.ForwardSensitivityOutOfPlaceError ReverseDiff.gradient(Reverse, senseloss(ForwardSensitivity()),
```

Comment:
Why an oop error? The problem is in position. It gives a different error. Is Reversediff + iip + ForwardSensitivity known to be broken, or should this work?

Reply:
I think this had more to do with ReverseDiff requiring AbstractArray returns, so reversediff is known to fail in this case (see it marked as failing in the rest of the file too)